settings/local.py

```diff
@@ -29,6 +29,15 @@
 EMAIL_BACKEND = env('DJANGO_EMAIL_BACKEND',
                     default='django.core.mail.backends.console.EmailBackend')
 
+# DATABASE CONFIGURATION
+# ------------------------------------------------------------------------------
+# See: https://docs.djangoproject.com/en/dev/ref/settings/#databases
+DATABASES = {
+    # Raises ImproperlyConfigured exception if DATABASE_URL not in os.environ
+    'default': env.db("DATABASE_URL", default="sqlite:///%s/fossevents.db" % ROOT_DIR),
+}
+DATABASES['default']['ATOMIC_REQUESTS'] = True
```

Comment:
Let's remove the DATABASE_URL settings from common to production.py at the same time, move the `psycopg2==2.6.1` requirement to `requirements.txt` thoughts?

psycopg2==2.6.1 has a dependency that requires postgres to be install on system before you can install the library. 

A note about this in the REAMDE setup section would help as well.